DEVELOPING.md

```diff
@@ -60,13 +60,13 @@ rule is that minors are breaking changes and patches are new features).
 1. May be necessary if Go code has changed though probably it was already necessary to run this to keep CI green: `bazel run :gazelle`
 1. If we depend on a newer rules_nodejs, update the `check_rules_nodejs_version` in `ts_repositories.bzl`
 1. `git commit -a -m 'Update docs for release'`
-1. `npm config set tag-version-prefix ''`
-1. `npm version minor -m 'rel: %s'` (replace `minor` with `patch` if no breaking changes)
-1. Build npm packages and publish them: `bazel run //internal:npm_package.publish && bazel run //internal/karma:npm_package.publish`
-1. `bazel build :release`
+1. Update `VERSION` in `version.bzl` (increment `patch` if no breaking changes, otherwise increment `minor`)
```

Comment:
let's do a separate commit on the release instructions after we successfully do a release and have refined the process

Reply:
👍